aws_xray_sdk/core/sampling/sampler.py

```diff
@@ -9,6 +9,7 @@
 from .target_poller import TargetPoller
 from .connector import ServiceConnector
 from .reservoir import ReservoirDecision
+import aws_xray_sdk
```

Comment:
should this import be updated?

Reply:
Ah good catch. I must've missed this one.